src/main/java/ui/windows/game/GameDisplayData.java

```diff
@@ -6,17 +6,23 @@ public class GameDisplayData {
     private String currentPlayer;
     private ArrayList<CardDisplayData> cards;
     private CardDisplayData currentCard;
+    private boolean hasWinner;
 
-    public GameDisplayData(String currentPlayer, ArrayList<CardDisplayData> cards, CardDisplayData currentCard) {
+    public GameDisplayData(String currentPlayer, ArrayList<CardDisplayData> cards, CardDisplayData currentCard, boolean hasWinner) {
         this.currentPlayer = currentPlayer;
         this.cards = cards;
         this.currentCard = currentCard;
+        this.hasWinner = hasWinner;
     }
 
     public String getCurrentPlayer() {
         return currentPlayer;
     }
 
+    public boolean getHasWinner() {
```

Comment:
The other methods should have documentation from #76, but please add some to this method. :)